_pages/business-units/18f/chapters/engineering.md

```diff
@@ -126,8 +126,7 @@ Most of the Engineering team works on a mix of partner agency projects and inter
 
 ### Facilitator groups
 
-[#eng-cronus1](https://gsa-tts.slack.com/messages/eng-cronus1), [#eng-chumanjalaal](https://gsa-tts.slack.com/messages/eng-chumanjalaal), [#eng-voyager1](https://gsa-tts.slack.com/messages/eng-voyager1), [#spaceball1](https://gsa-tts.slack.com/messages/spaceball1), [#eng-apollo](https://gsa-tts.slack.com/messages/eng-apollo), [#eng-bellatrix](https://gsa-tts.slack.com/messages/eng-bellatrix), [#eng-eagle5](https://gsa-tts.slack.com/messages/eng-eagle5),
```

Comment:
I am very disappointed we did not, at one time, have a group called `eng-cronuts`.